### PR TITLE
DDCNL-4271

### PR DESCRIPTION
**Pull request by andrewbrown22**

### CODE REVIEW

File: app/uk/gov/hmrc/nationalinsurancerecord/connectors/DesConnector.scala [hunk 1]
```diff
@@ -18,6 +18,7 @@ package uk.gov.hmrc.nationalinsurancerecord.connectors
 
 import com.google.inject.Inject
 import play.api.Logger
```
Comment (Tateni93) [inline app/uk/gov/hmrc/nationalinsurancerecord/connectors/DesConnector.scala:20]: Do we still need this import if we are now extending Logger

File: app/uk/gov/hmrc/nationalinsurancerecord/controllers/ErrorHandling.scala [hunk 2]
```diff
@@ -17,32 +17,32 @@
 package uk.gov.hmrc.nationalinsurancerecord.controllers
 
 import play.api.Logger
```
Comment (Tateni93) [inline app/uk/gov/hmrc/nationalinsurancerecord/controllers/ErrorHandling.scala:19]: See above

**Review by Tateni93 — APPROVED** [review]

File: app/uk/gov/hmrc/nationalinsurancerecord/connectors/DesConnector.scala (hunk 1 above)
Comment (andrewbrown22) [reply app/uk/gov/hmrc/nationalinsurancerecord/connectors/DesConnector.scala]: removed

File: app/uk/gov/hmrc/nationalinsurancerecord/controllers/ErrorHandling.scala (hunk 2 above)
Comment (andrewbrown22) [reply app/uk/gov/hmrc/nationalinsurancerecord/controllers/ErrorHandling.scala]: removed

**Pull request merged**